### PR TITLE
Repair some of the new phpstan errors

### PR DESCRIPTION
**Pull request by Neloop**

### CODE REVIEW

File: app/V1Module/security/AccessManager.php [hunk 1]
```diff
@@ -74,12 +73,6 @@ public function decodeToken($token): AccessToken
             throw new InvalidAccessTokenException($token);
         } catch (UnexpectedValueException $e) {
             throw new InvalidAccessTokenException($token);
-        } catch (ExpiredException $e) {
-            throw new InvalidAccessTokenException($token);
-        } catch (SignatureInvalidException $e) {
-            throw new ForbiddenRequestException();
```
Comment (Neloop) [inline app/V1Module/security/AccessManager.php:80]: Damn... this is not correct

File: app/helpers/PermissionHints.php [hunk 2]
```diff
@@ -19,15 +19,15 @@ public function __construct()
...
+     * @param object $aclModule An ACL module
+     * @param object $subject The resource checked for permissions
      * @return Generator
      */
     public static function generate($aclModule, $subject)
     {
         foreach (static::getAclMethods($aclModule) as $method) {
             $parameter = $method->getParameters()[0];
-            $className = $parameter->getClass() ? $parameter->getClass()->getName() : null;
+            $className = $parameter->getClass()->getName();
```
Comment (Neloop) [inline app/helpers/PermissionHints.php:30]: This is probably also not correct...

File: app/V1Module/security/AccessManager.php (hunk 1 above)
Comment (Neloop) [reply app/V1Module/security/AccessManager.php:80]: Repaired

File: app/helpers/PermissionHints.php (hunk 2 above)
Comment (Neloop) [reply app/helpers/PermissionHints.php]: This is actually bad typehint by nette... changed the code back

File: app/V1Module/security/UserStorage.php [hunk 3]
```diff
@@ -35,12 +35,13 @@ public function __construct(AccessManager $accessManager, Nette\Http\IRequest $h
     }
 
     /**
-     * @param bool
-     * @return void
+     * @param bool $state
+     * @return static
```
Comment (krulis-martin) [inline app/V1Module/security/UserStorage.php:39]: static?

File: app/commands/GenerateSwagger.php [hunk 4]
```diff
@@ -122,15 +122,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if (!$apiRoutes) {
             $output->writeln("<error>No suitable routes found</error>");
-            return;
+            return 1;
```
Comment (krulis-martin) [inline app/commands/GenerateSwagger.php:125]: What is this new devilry?

Comment (Neloop) [reply app/commands/GenerateSwagger.php:125]: Implementation of `execute` method of `Command` should return integer.

See https://github.com/symfony/console/blob/5.x/Command/Command.php#L163

File: app/V1Module/security/UserStorage.php (hunk 3 above)
Comment (Neloop) [reply app/V1Module/security/UserStorage.php]: Changed to `self` which is more self-explaining

**Review by krulis-martin — APPROVED** [review]

**Pull request merged**